packages/react-environment/src/api-providers/web3/webb-web3-provider.ts

```diff
@@ -110,4 +118,21 @@ export class WebbWeb3Provider
   endSession(): Promise<void> {
     return this.web3Provider.endSession();
   }
+
+  async reason(hash: string) {
```

Comment:
can this function be called 'txFailureReason' or something more descriptive of the failure?

Reply:
this didn't work actually still need to hack with it will keep it for reference in the meantime